File: SignalRDashboard.Data.Milliman/Hubs/Models/ConfidenceStatus.cs
using SignalRDashboard.Data.Core.Hubs.Models;

namespace SignalRDashboard.Data.Milliman.Hubs.Models
{
    public class ConfidenceStatus : DashboardHubModel
    {
        private decimal _teamCityConfidence;
        public decimal TeamCityConfidence
        {
            get { return _teamCityConfidence; }
            set { SetProperty(ref _teamCityConfidence, value); }
        }
    }
}

File: SignalRDashboard.Web/Scripts/app/models/milliman/ConfidenceStatus.js
(function(signalrdashboardMilliman) {
    signalrdashboardMilliman.ConfidenceStatus = ConfidenceStatus;

    function ConfidenceStatus() {
        // Properties matching data model received via SignalR
        this.teamCityConfidence = '';

        // Error check conditions
        var model = this;
        var errorChecks = [];

        this.updateFromData = function(data) {            
            signalrdashboard.mapping.map(data, this);
            signalrdashboard.alertHandler.performErrorChecks('ConfidenceStatus', errorChecks);
        };
    }
    
})(window.signalrdashboard.milliman || (window.signalrdashboard.milliman = {}));